next/tsconfig.json

```diff
@@ -30,5 +30,5 @@
     }
   },
   "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx", "next-env.d.ts"],
-  "exclude": ["node_modules", "schema-generator"]
+  "exclude": ["node_modules", "schema-generator", "./cypress.config.ts", "cypress"]
```

Comment:
Cypress causing type errors in Jest assertions. So there was a problem with failing the typescript check.

Reply:
This does not look right, if you exclude the entire cypress directory and the tsconfig inside of it extends this config, then I don't think the files would be correctly processed by the editor ? At least my local VS Code complains in every file I open form the cypress directory:

<img width="571" alt="Screenshot 2023-11-15 at 15 42 32" src="https://github.com/bratislava/konto.bratislava.sk/assets/2327312/801338bd-bad5-40a4-876b-2c227a4030df">